packages/block-editor/src/components/inserter/block-list.js

```diff
@@ -186,6 +189,10 @@ function InserterBlockList( {
 
 	const hasItems = ! isEmpty( filteredItems );
 	const hasChildItems = childItems.length > 0;
+	const slot = { fills: [ 'test' ] }; //useSlot( __experimentalInserterMenuExtension.Slot.slotName );
```

Comment:
For some reason,  uncommenting the hook here breaks the editor (you need to enable "block directory search" experiment to experience it). I'm not sure yet why, it seems it re-renders continuously. cc @diegohaz in case you have an idea here?

Reply:
Fix this one, it looks like the fillProps for slots need to be memoized otherwise we risk infinite loops. For now, that's what I did but maybe there's a better fix for that later.